### ISSUE TITLE
Install with pipenv fails for version 1.0.3

### ISSUE THREAD

**Issue by AndreKkl** [Unidata/cftime#99]
On Debian in a Dockerfile with Pipfile:

RUN pip install pipenv && pipenv install
with cftime = "*" in Pipfile
results in:
`pipenv.patched.notpip._internal.exceptions.InstallationError: Command "python setup.py egg_info" failed with error code 1 in /tmp/tmp3dy86zfkbuild/cftime/`

Version 1.0.0 or also 1.0.2.1 does not have this problem.

Installation with pip install cftime works for both versions.

**Comment by gdetrez** [comment]
I have a similar issue when trying to install cftime in a virtualenv:
```
❯ ./tmpenv/bin/pip install --no-cache cftime==1.0.3
Collecting cftime==1.0.3
  Downloading https://files.pythonhosted.org/packages/80/23/cd5513106c709f1faa3623bb34a5284b55ea7dad4229103b68c154d8e39f/cftime-1.0.3.tar.gz (45kB)
    100% |████████████████████████████████| 51kB 3.0MB/s 
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/gregoire/pip-install-3nbyhbg0/cftime/setup.py", line 6, in <module>
        from Cython.Build import cythonize
    ModuleNotFoundError: No module named 'Cython'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/gregoire/pip-install-3nbyhbg0/cftime/
```

It works if I install Cython first (`pip install cython`).  This might be the same issue as #100 (missing wheels on pypi).

**Comment by AndreKkl** [comment]
@gdetrez I can confirm that it's workign when I make  
`pipenv` install Cython && pipenv install`
However, this should not be necessary as it goes against the principle of a pipfile

**Comment by jswhit** [comment]
There will be a bug fix release (1.0.3.1) shortly to fix this.
